modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+...
+----
+
+Each log stream contains log events. Here, to see a log event from the `busybox` Pod, you specify a log stream from the `application` log group:
+
+[source,terminal]
+----
+$ aws logs get-log-events --log-group-name mycluster-7977k.application --log-stream-name kubernetes.var.log.containers.busybox_app_busybox-da085893053e20beddd6747acdbaf98e77c37718f85a7f6a4facf09ca195ad76.log
+{
+    "events": [
+        {
+            "timestamp": 1629422704178,
+            "message": "{\"docker\":{\"container_id\":\"da085893053e20beddd6747acdbaf98e77c37718f85a7f6a4facf09ca195ad76\"},\"kubernetes\":{\"container_name\":\"busybox\",\"namespace_name\":\"app\",\"pod_name\":\"busybox\",\"container_image\":\"docker.io/library/busybox:latest\",\"container_image_id\":\"docker.io/library/busybox@sha256:0f354ec1728d9ff32edcd7d1b8bbdfc798277ad36120dc3dc683be44524c8b60\",\"pod_id\":\"870be234-90a3-4258-b73f-4f4d6e2777c7\",\"host\":\"ip-10-0-216-3.us-east-2.compute.internal\",\"labels\":{\"run\":\"busybox\"},\"master_url\":\"https://kubernetes.default.svc\",\"namespace_id\":\"794e1e1a-b9f5-4958-a190-e76a9b53d7bf\",\"namespace_labels\":{\"kubernetes_io/metadata_name\":\"app\"}},\"message\":\"My life is my message\",\"level\":\"unknown\",\"hostname\":\"ip-10-0-216-3.us-east-2.compute.internal\",\"pipeline_metadata\":{\"collector\":{\"ipaddr4\":\"10.0.216.3\",\"inputname\":\"fluent-plugin-systemd\",\"name\":\"fluentd\",\"received_at\":\"2021-08-20T01:25:08.085760+00:00\",\"version\":\"1.7.4 1.6.0\"}},\"@timestamp\":\"2021-08-20T01:25:04.178986+00:00\",\"viaq_index_name\":\"app-write\",\"viaq_msg_id\":\"NWRjZmUyMWQtZjgzNC00MjI4LTk3MjMtNTk3NmY3ZjU4NDk1\",\"log_type\":\"application\",\"time\":\"2021-08-20T01:25:04+00:00\"}",
+            "ingestionTime": 1629422744016
+        },
+...
+----
+
+.Example: Replacing the default `infrastructureName` prefix in log group names
+
+You can replace the default `infrastructureName` prefix, `mycluster-7977k`, in the log group names by specifying a `groupPrefix` field in the `ClusterLogForwarding` CR:
+
+[source,yaml]
+----
+cloudwatch:
+    region: us-east-2
+    groupBy: logType
+    groupPrefix: demo-group-prefix
+----
+
+The value of `groupPrefix` replaces the default `infrastructureName` prefix:
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-groups | jq .logGroups[].logGroupName
+"demo-group-prefix.application"
+"demo-group-prefix.audit"
+"demo-group-prefix.infrastructure"
+----
+
+.Example: Naming log groups for application namespace names
+
+For each application namespace, you can create a log group whose name is based on the name of the application namespace.
+
+If you destroy a namespace object and create a new one that has the same application namespace name, CloudWatch continues using the same log group as before.
```

Comment:
`destroy` is a little stronger. Consider more neutral language. Something like, `If you delete/remove a namespace...`

Also, to avoid ending the sentence with a preposition, consider rephrasing to, `...CloudWatch continues using the previous log group.`